.github/CODEOWNERS

```diff
@@ -67,6 +67,7 @@
 /ilert/                                  @yacut support@ilert.com
 /insightfinder/                          @dearmore @erinlmcmahon support@insightfinder.com
 /rum_javascript/                         @DataDog/rum-app
+/kamailio/                               @devopsec
```

Comment:
Could you also add `kamailio` to [this section](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L169) so our doc team will be notified when needed?

Reply:
resolved in cf85a7a2cd46cc4e6c399e639ff477102a24aead